src/H5.c

```diff
@@ -816,13 +791,10 @@ H5get_libversion(unsigned *majnum /*out*/, unsigned *minnum /*out*/, unsigned *r
 } /* end H5get_libversion() */
 
 /*-------------------------------------------------------------------------
- * Function:    H5check_version
+ * Function:    H5__check_version
```

Comment:
Extracted core of the checking into an internal routine that can be called from several places.